lib/features.rb

```diff
@@ -0,0 +1,47 @@
+class Features
+  def all
+    prod = get('www')
```

Comment:
```suggestion
    prod = get_feature_flags_for('www')
```

Or something along those lines, makes it a bit more readable.

Reply:
Done!